python/ray/data/_internal/iterator/stream_split_iterator.py

```diff
@@ -204,7 +208,7 @@ def get(
 
         This is intended to be called concurrently from multiple clients.
         """
-
+        start_time = time.perf_counter()
```

Comment:
nit, consider creating a context manager or decorator util to make the code cleaner.

Reply:
I considered that, I think best to leave as is in case the stats object is `None`. I think for my previous PR switching to a context manager caused some integration tests to fail (https://github.com/ray-project/ray/pull/43283). I could use the context manager only when stats is not `None` but not sure if that would end up being cleaner.